### PR TITLE
Skip schema prefix when dialect is SQLite

### PR DESCRIPTION
**Pull request by marwan-at-work**
SQLite does not have schema prefixes when referencing a table, make sure the Builders account for this case.

### CODE REVIEW

File: dialect/sql/builder.go [hunk 1]
```diff
@@ -1819,6 +1819,7 @@ func (s *Selector) Query() (string, []interface{}) {
 	b.WriteString(" FROM ")
 	switch t := s.from.(type) {
 	case *SelectTable:
+		fmt.Println("ONE", s.dialect)
```
Comment (a8m) [inline dialect/sql/builder.go:1822]:
```suggestion
```

Remove please.

File: dialect/sql/builder.go [hunk 2]
```diff
@@ -682,7 +682,7 @@ func (i *InsertBuilder) Returning(columns ...string) *InsertBuilder {
 // Query returns query representation of an `INSERT INTO` statement.
 func (i *InsertBuilder) Query() (string, []interface{}) {
 	i.WriteString("INSERT INTO ")
-	if i.schema != "" {
+	if i.schema != "" && i.dialect != dialect.SQLite {
 		i.Ident(i.schema).WriteByte('.')
 	}
```
Comment (a8m) [inline dialect/sql/builder.go:687]: Maybe moving this code block to the `Builder`, and then calling it from all builders as follows:

```go
func (b *Builder) writeSchema(s schema)  {
	if schema != "" && u.dialect != dialect.SQLite {
		b.Ident(u.schema).WriteByte('.')
	}
}

// ..

u.writeSchema(u.schema)
```

Comment (marwan-at-work) [reply dialect/sql/builder.go]: @a8m done 👍🏼

**Review by a8m — APPROVED** [review]
Thanks for your contribution @marwan-at-work 😃

**Pull request merged**